JetAnalyzers/bin/jet_response_fitter_x.cc

```diff
@@ -138,21 +145,14 @@ int main(int argc,char**argv)
     
     if (algs.size()>0&&!contains(algs,alg)) continue;
 
```

Comment:
I also meant lines 141-145 in the original version of the diff. It warns the user and continues if a directory already exists. Now that update is back this section is necessary as well. BTW, this current functionality saved my butt yesterday, so I know it is useful.